docs/en/latest/config.json

```diff
@@ -78,7 +78,8 @@
             "plugins/ip-restriction",
             "plugins/ua-restriction",
             "plugins/referer-restriction",
-            "plugins/consumer-restriction"
+            "plugins/consumer-restriction",
+            "plugins/opa"
```

Comment:
IMHO, it should be put into Authentication as opa is similar to casbin/keycloak

Reply:
changed